src/helpers/MoviesHelper.js

```diff
@@ -0,0 +1,36 @@
+import Config from 'react-native-config';
```

Comment:
Instead of Helper what do you think about naming it something like "Service" or "Client" so it is more descriptive of what it does, in the template I think these kind of files are inside "Controllers"

Reply:
Oh, that totally makes sense. I'll move it to the controllers with a proper name. Thanks!